src/components/CurrentCityForecast/components/DailyForecasts/DailyForecasts.jsx

```diff
@@ -0,0 +1,19 @@
+import classes from './DailyForecasts.module.scss';
+
+import DailyForecast from './DailyForecast/DailyForecast';
+
+function DailyForecasts() {
+  return (
+    <div className={classes.forecastsContainer}>
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
+      <DailyForecast />
```

Comment:
Will you replace this to a loop in a future?

Reply:
Yes. I will do it, when I will connect this component with API.